tools/docker/env

```diff
@@ -0,0 +1,21 @@
+#!bash
+# fail fast when something goes wrong
+set -e
+
+SCRIPT_DIR=$( cd -- "$( dirname -- "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )
+REPO_DIR="$SCRIPT_DIR/../.."
+CACHE_DIR="$REPO_DIR/.cache"
+
+mkdir -p "$CACHE_DIR"
+
+docker build -t myimage \
```

Comment:
This shouldn't be naming it something (at least as trivial as `myimage`), and should instead reference the freshly built image via it's tag.

Likely convert this to `docker build -q`, though I will miss the output in case of failures.